### PR TITLE
Added documentation to the home package

### LINKED ISSUE #181: Document the files in mifopay/home package
**Summary:**

Document the files in mifopay/home package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixes #181 

**Please Add Summary of what changes you have made.(Optional)**
Added documentation to Home Package

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by codepoet2017390** [comment]
@ankurs287 @luckyman20  please review

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java [hunk 1]
```diff
@@ -19,6 +19,11 @@
 import butterknife.BindView;
 import butterknife.ButterKnife;
 
+/**
+ * PaymentsFragment
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java]:
```suggestion
 * Payments Fragment
```

**Comment by codepoet2017390** [comment]
@luckyman20 updated

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/adapter/TabLayoutAdapter.java [hunk 2]
```diff
@@ -7,30 +7,55 @@
...
+     * Returns Size of the FragmentList.
+     */
     @Override
     public int getCount() {
         return mFragmentList.size();
     }
 
+    /**
+     * Returns title of the Fragment at specified position.
+     * @param position position of the Fragment whose title is required
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/adapter/TabLayoutAdapter.java]:
```suggestion
     * @param position Position of the Fragment whose title is required
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java [hunk 3]
```diff
@@ -160,17 +179,28 @@ public void onSlide(@NonNull View view, float v) {
...
+     */
     @Override
     public void setPresenter(BaseHomeContract.HomePresenter presenter) {
         this.mHomePresenter = presenter;
     }
 
+    /**
+     * Updates the text view with accountBalance and currency symbol.
+     * @param account instance of Account class.
+     *                is specific to the a user.
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: Keep it in one line only.

**Comment by codepoet2017390** [comment]
@luckyman20  updated

**Comment by shiv07tiwari** [comment]
Hey @codepoet2017390 , there has been a lot of changes in the project and thus we need a fresh start for the documentation part. :smile:

**Pull request closed without merging**